help/md/dnPhyloCTMC.md

```diff
@@ -1,9 +1,218 @@
 ## name
-dnPhyloCTMC
+`dnPhyloCTMC`: Distribution of a phylogenetic continuous-time Markov chain
+
 ## title
+The parameters of a phylogenetic model – a tree topology with branch lengths, a substitution model that describes how observations evolve over the tree, etc. – collectively form a distribution called the _phylogenetic continuous-time Markov chain_.
+
 ## description
+dnPhyloCTMC gives the probability distribution of tip labels on a phylogenetic tree given an phylogenetic continuous-time Markov chain model.
+
 ## details
+
+The likelihood of observed tip labels (specified via a clamped `AbstractHomologousDiscreteCharacterData` object) is computed using Felsenstein's pruning algorithm, with partial likelihoods stored for each branch of the tree. It is automatically outputted in the `Likelihood` column of the `mnFile()` and `mnScreen()` monitors (which can be suppressed with `likelihood = FALSE`).
+
 ## authors
 ## see_also
+- Tutorial on [graphical models](https://revbayes.github.io/tutorials/intro/graph_models)
+
+- Tutorial on [specifying a phylogenetic continuous-time Markov chain](https://revbayes.github.io/tutorials/ctmc/) model
+
 ## example
+
+```rb
+# Create stochastic node with the tip distribution given by the topology `tree` with the 
+# rate matrix parameter q_matrix
+x ~ dnPhyloCTMC(tree = tree, Q = q_matrix)
+
+# Clamp observed characters to the node
+chars <- readDiscreteCharacterData("myData.nex")
+x.clamp(chars)
+```
+
+Simpler distributions contain methods to compute the probability of a particular value
+```rb
+x ~ dnNormal(0.0, 1.0)
+x.clamp(0.42)
+x.lnProbability()
+```
+
+The likelihood of observed characters given fixed parameter values (tree with branch lengths, site rates, etc.)
+can be computed by running a dummy single-iteration MCMC analysis:
+
+```rb
+mymodel = model(tree) # will incoroporate all phyloCTMC objects with `tree` as a parameter
+monitors = VectorMonitors()
+monitors.append( mnScreen(printgen=1) )
+mymcmc = mcmc(mymodel, monitors, moves)
+mymcmc.run(generations=1)
+```
+
+The value is given in the Likelihood column of the first (i.e. 0th-iteration) row.
+
+
+## Methods
```

Comment:
This list mixes up actual methods of `dnPhyloCTMC()`, like `clamp`, `probability`, and `redraw`, with methods of `AbstractHomologousDiscreteCharacterData` – i.e., methods that cannot be applied to the distribution itself, but only to draws from it. This is not accurate.